src/GeoJson/CoordinateReferenceSystem/CoordinateReferenceSystem.php

```diff
@@ -90,15 +78,7 @@ final public static function jsonUnserialize($json)
     /**
      * Factory method for creating a CRS object from properties.
      *
-     * This method must be overridden in a child class.
-     *
-     * @see CoordinateReferenceSystem::jsonUnserialize()
      * @param array|object $properties
-     * @return CoordinateReferenceSystem
-     * @throws \BadMethodCallException
      */
-    protected static function jsonUnserializeFromProperties($properties)
-    {
-        throw new \BadMethodCallException(sprintf('%s must be overridden in a child class', __METHOD__));
-    }
```

Comment:
Although I don't expect this class is extended beyond the Linked and Named classes within this library, changing this to abstract may constitute a BC break since an error would now be raised at interpretation time instead of runtime. Can this be reverted?

I expect this won't even be relevant if the entire class is removed in 2.0.

Reply:
It's good that you said that, I wasn't sure myself (going back and forth between 1.x and 2.0). I reverted the change 👍